jrnl/jrnl.py

```diff
@@ -169,6 +172,16 @@ def search_mode(args, journal, **kwargs):
     # Where do the search results go?
     if args.edit:
         _edit_search_results(**kwargs)
+        # If we want to both edit and change time in one action
+        if args.change_time:
+            # Re-filter the journal enties (_edit_search_results puts
```

Comment:
This is a good observation. We should (in some other PR, not this one) separate the filtering and recombining of entries so that each of these commands doesn't have to worry about it. Thanks for pointing this out.